docker/include/bin/start_faux

```diff
@@ -134,7 +134,44 @@ echo Faux device at $local_ip bcast $broadcast_ip
 # Setup fake telnet listener to trigger port-scan failure
 if [ -n "${options[telnet]}" ]; then
     echo Enabling mock telnet server...
-    (while true; do echo Telnet `hostname`; nc -nvlt -p 23 -e `which hostname`; done) &
+    while true; do echo -e "Telnet $(hostname)" | nc -l -w 1 23; done)&
+fi
+
+if [ -n "${options[oddservices]}" ]; then
+    echo Running services on non standard ports and open default ports 
+
+    echo Starting FTP 21514 and open default 20,21
+    nc -nvlt -p 20&
+    nc -nvlt -p 21&
+    (while true; do echo -e "220 ProFTPD 1.3.5e Server (Debian) $(hostname)" | nc -l -w 1 21514; done)&
+    
+    echo Starting SMTP 1256 and open default 25, 465, 587
+    nc -nvlt -p 25&
+    nc -nvlt -p 465&
+    nc -nvlt -p 587&
+    (while true; do echo -e "220 $(hostname) ESMTP Postfix (Ubuntu)" | nc -l -w 1 1256; done)&
+
+    echo Starting IMAP 5361 and open default ports 143, 993
+    nc -nvlt -p 143&
+    nc -nvlt -p 993&
+    (while true; do echo -e "* OK [CAPABILITY IMAP4rev1 LITERAL+ SASL-IR LOGIN-REFERRALS ID ENABLE IDLE STARTTLS AUTH=PLAIN] Dovecot (Ubuntu) ready.\r\n" \
+        | nc -l -w 1 5361; done)&
+
+    #echo Starting TELNET and default port 23
```

Comment:
restore or remove

Reply:
Removed and existing telnet faux option reinstated